### PR TITLE
Maxscale role installation/configuration and tests

### PR DESCRIPTION
**Pull request by gfaivre**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Change Log
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/)
+and this project adheres to [Semantic Versioning](http://semver.org/).
+
+## [Unreleased]
+
+## [1.0.0] - 2017-02-14
+
```
Comment (nervo) [inline CHANGELOG.md]: No cr here :)

File: tests/030_configs.retry [hunk 2]
```diff
@@ -0,0 +1 @@
+jessie
```
Comment (nervo) [inline tests/030_configs.retry]: Missing .gitignore with
```
/tests/*.retry
```

**Comment by nervo** [comment]
Tell me if i'm wrong, but as there is *only one* config file, you should use "manala_maxscale_config" keys, and not "manala_maxscale_configs".
Have a look on the nginx role, where we are using both.

**Comment by nervo** [comment]
👍

File: README.md [hunk 3]
```diff
@@ -8,3 +8,105 @@ It's part of the [Manala Ansible stack](http://www.manala.io) but can be used as
...
+
+| Name             | Type    | Description              |
+| ---------------- | ------- | ------------------------ |
+| maxscale restart | Service | Restart Maxscale service |
+
+## Role Variables
+
+| Name                                | Default           | Type          | Description                                            |
+| ----------------------------------- | ----------------- | ------------- | ------------------------------------------------------ |
+| `manala_maxscale_config_file`       | /etc/maxscale.cnf | String (path) | Configuration file                                     |
```
Comment (hyvs) [inline README.md]:
```diff
--- String (path)
+++ String
```

File: README.md [hunk 4]
```diff
@@ -8,3 +8,105 @@ It's part of the [Manala Ansible stack](http://www.manala.io) but can be used as
...
+| `manala_maxscale_config_template`   | config/empty.j2   | String (path) | Default configuration template                         |
+| `manala_maxscale_config`            | [ ]               | Array         | Maxscale configuration options                         |
```
Comment (hyvs) [inline README.md]:
```diff
- [ ]
+ []
```

**Review by hyvs — CHANGES_REQUESTED** [review]
Just some syntax fixes in Readme. Nice job

**Comment by gfaivre** [comment]
Done.

**Review by hyvs — APPROVED** [review]

**Pull request merged**